gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/writer/HiveMetadataWriter.java

```diff
@@ -138,7 +138,9 @@ public void flush(String dbName, String tableName) throws IOException {
       //iterator all execution to get the result to make sure they all succeeded
       for (HashMap.Entry<List<String>, ListenableFuture<Void>> execution : executionMap.entrySet()) {
         try {
+          log.debug("Start registering partition during hive write for db: " + dbName + " table: " + tableName + " partition: " + Joiner.on(",").join(execution.getKey()));
           execution.getValue().get(timeOutSeconds, TimeUnit.SECONDS);
```

Comment:
As this is ascync call, even we don't call get here, the hive registration might be executed as well, and in that case, we may miss the event